GETTING_STARTED.md

```diff
@@ -387,7 +387,7 @@ post.new_record?        # => true
 post.author.new_record? # => false
 ```
 
-To not save the associated object, specify strategy: :build in the factory:
+To not save the associated object, specify `strategy: :build` in the factory:
```

Comment:
It is marked as code also at the following place:
https://github.com/thoughtbot/factory_bot/pull/1307/files#diff-df17b1aeaf5e7d797809ea45cc1271acR406